src/components/home.tsx

```diff
@@ -1,78 +1,86 @@
+        <Heading as="h2" size="large">
+          No polls
+        </Heading>
+      </Flex>
+    );
+  }
+  return (
+    <>
+      <Heading as="h2" size="large" mb={5}>
+        Welcome {user?.name}
+      </Heading>
+      <Box>
+        <Grid templateColumns="repeat(auto-fill, minmax(300px, 1fr))" gap="5">
+          {polls.map(({ id, question, poll_count }) => (
+            <Link to={`/poll/${id}`}>
+              <Box
+                background="#f2f9e3"
+                padding={3}
+                borderRadius={5}
+                fontWeight="bold"
+              >
+                <p>{question}</p>
+                <p>{poll_count}</p>
+              </Box>
+            </Link>
+          ))}
+        </Grid>
+      </Box>
+    </>
+  );
+};
 
-    useEffect(() => {
-        (async () => {
-            const res = await fetch('https://tva-backend.herokuapp.com/polls');
-            const data = await res.json();
-            setPolls(data);
-        })();
-    }, []);
+interface HomeProps {}
 
-    const displayPolls = () => {
-        return polls.map(poll => {
-            const { id, question, poll_count } = poll;
+const Home: FunctionComponent<HomeProps> = () => {
+  const [polls, setPolls] = useState<IPoll[]>([]);
 
-            return (
-                <Link to={`/poll/${id}`}>
-                    <Box
-                        background="#f2f9e3"
-                        padding={3}
-                        borderRadius={5}
-                        fontWeight="bold"
-                    >
-                        <p>{question}</p>
-                        <p>{poll_count}</p>
-                    </Box>
-                </Link>
-            )
-        })
-    }
+  useEffect(() => {
+    (async () => {
+      const res = await fetch("https://tva-backend.herokuapp.com/polls");
+      const data = await res.json();
+      setPolls(data);
+    })();
+  }, []);
 
-    return (
-        <>
-            {auth.user ? (
-                <Box>
-                    <Heading as="h2" size="large" mb={5}>Welcome {auth.user}</Heading>
-                    <Box>
-                        {polls.length ?
-                            <Grid
-                                templateColumns="repeat(auto-fill, minmax(300px, 1fr))"
-                                gap="5"
-                            >
-                                {displayPolls()}
-                            </Grid>
-                            :
-                            <Flex height="65vh">
-                                <Heading as="h2" size="large">No polls</Heading>
-                            </Flex>
-                        }
-                    </Box>
-                </Box>
-            ) : (
-                <Box>
-                    <Heading as="h2" size="large">You must be logged in. Please <Link style={{ textDecoration: 'underline' }} to="/login">Log in</Link>.</Heading>
-                </Box>
-            )}
-        </>
-    );
-}
+  return (
```

Comment:
Using the conditional rendering with fallback here for Authentication. The `PollsList` should be it's own component with it's own responsibility. 

I also recommend writing helper functions inside components that return the same thing as a component could return. Instead of a function called `displayPolls` I recommend a component called `PollsList`.